util/test/checkChplInstall

```diff
@@ -25,16 +25,30 @@
 # To use something other than $HOME/.chpl, set the CHPL_CHECK_INSTALL_DIR
 # environment variable.
 
+function log_date()
+{
+    echo -n "$(date '+%Y-%m-%d %H:%M:%S') "
+}
 
-# Base name for test job in $CHPL_HOME/test/release/examples
-TEST_JOB_BASENAME=hello6-taskpar-dist
+function log_info()
+{
+    local msg=$@
+    log_date
+    echo "[INFO] ${msg}"
+}
 
-err() {
-    echo "error: $@" >&2
+function log_error()
+{
+    local msg=$@
+    log_date
+    echo "[ERROR] ${msg}" >&2
 }
 
+# Base name for test job in $CHPL_HOME/test/release/examples
```

Comment:
Not new, but this comment is slightly off since the test is in a difference place for the tarball